templates/collectd_metrics/template.yml

```diff
@@ -0,0 +1,11 @@
+skeleton_path: ../skeleton.json
+
+elasticsearch_template:
+  name: com.ovirt.viaq-collectd
+  index_pattern: "*"
+  order: 10
+
+namespaces:
+  - collectd.yml
+  - pipeline_metadata.yml
+#  - openshift/app_log.yml
```

Comment:
We can drop this, unless this will need to be uncommented before this is committed?